src/util.ts

```diff
@@ -54,7 +56,15 @@ export const PROVIDER_ERRORS = {
   }),
 };
 
-const getHash = async (data: ArrayBuffer, useNative: boolean): Promise<any> => {
+const getHash = async (
+  data: ArrayBuffer,
+  nativeCrypto?: NativeCrypto,
+  useNative = true,
```

Comment:
It feels a bit confusing to have both the notion of `nativeCrypto` and a `useNative` flag that don't mean the same thing here